src/psyclone/psyir/tools/read_write_info.py

```diff
@@ -100,12 +101,27 @@ def write_list(self):
     def signatures_written(self):
         '''Convenience function to return only the signatures written.
 
-        :returns the list of all signatures written.
+        :returns: the list of all signatures written.
         :rtype: List[:py:class:`psyclone.core.Signature`]
 
         '''
         return [sig for _, sig in self.write_list]
 
+    # -------------------------------------------------------------------------
+    @property
+    def signatures_readwrite(self):
```

Comment:
This may also not be needed if using the self.reference_access but I don't have any problem with it if its useful for other cases.

My only issue is that it feels the same as the functionality in SingleVariableAccessInfo and I am wondering if we are duplicating things.

Reply:
Reverted.